updatecli/updatecli.d/agent.yaml

```
@@ -0,0 +1,33 @@
+sources:
+  lastGithubRelease:
+    kind: githubrelease
+    spec:
+      owner: 'grafana'
+      repository: 'agent'
+      token: '{{ requiredEnv "UPDATECLI_GITHUB_TOKEN" }}'
+      versionfilter:
+        kind: latest
```

```suggestion
        kind: semver
```
Nitpick for the future

The grafana/agent artifact seems to follow semantic versioning so you probably always want the latest version from a semantic versioning stand point instead of the "latest" released.